src/game/engine/radio.h

```diff
@@ -48,21 +49,21 @@ class RadioClass : public MissionClass
         RadioMessageType message, target_t &target = RadioClass::Get_LParam(), RadioClass *radio = nullptr);
     virtual RadioMessageType Transmit_Message(RadioMessageType message, RadioClass *radio);
 
-    // TODO, rename!
-    bool const Radio_Valid() const { return m_Radio != nullptr; }
+    bool const In_Contact() const { return m_Radio != nullptr; }
```

Comment:
Use BOOL for now to be consistent, also it should just be BOOL return, not BOOL const which has no effect on standard types. The method itself should remain const of course.